### ISSUE TITLE
HTML of Keyboard Shortcuts Dialog - empty li's, too many p's and div's

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#391]
While testing, I noticed some weird spacing in the A11y View, so I inspected the HTML for the keyboard  shortcuts dialog.
**Screen shot from inspector**
![screen shot 2018-04-10 at 12 29 29](https://user-images.githubusercontent.com/1902116/38565175-3624a974-3cbb-11e8-8a8d-49e840946ea9.png)

A `div` element cannot be direct child of an `ul` element

It is ok if `div` elements contain things that need to stick together in the PDOM. It's important that they do not come between natural parent and child elements.

The lists in the Keyboard Shortcuts dialog should be like this:
```html
<h2>Move or Jump Grabbed Balloon</h2>
<ul>
  <li>Move grabbed balloon up, left, down, or right with Arrow keys or with letter keys W, A, S, or D.</li>
  <li>Move slower with shift plus Arrow keys or Shift plus letter keys W, A, S, or D.</li>
  <li>Jump close to sweater with J plus S.</li>
  <li>Jump to wall with J plus W.</li>
  <li>Jump to near wall with J plus N.</li>
  <li>Jump to center with J plus C.</li>
</ul>
```
**Here's the ideal fix:**
- The API should require that an `li` element is required to be a direct child of its parent `ul`
- Use the `li` tag itself to wrap the content of the list item, rather than adding an extra `p` tag.
- If you must use a `p` tag, it needs to go inside the `li`.
- Same for the `div` elements. If possible, do not add extra `div`s. If not possible, the `div` and/or the `p` have to be inside the `li`.
- If the purpose of the `div` is to "keep related content together for placing in the PDOM", it would be better to contain the parent `ul`, or contain the parent `ul` together with its labeling heading than to contain individual list items.

Additionally, removing the `p` should improvement the visual layout of the the bulleted lists in the A11y View:
![screen shot 2018-04-10 at 12 58 53](https://user-images.githubusercontent.com/1902116/38566722-f3b8689c-3cbe-11e8-8bd3-ef8a4c5660c3.png)

**Comment by terracoda** [comment]
This sim is in Dev Testing, so tagging that issue

**Comment by terracoda** [comment]
@jessegreenberg, if you need me to provide a full example of the Keyboard Help dialog HTML, I can do that.

**Comment by jessegreenberg** [comment]
Thanks @terracoda, I shouldn't need a full example, I think you already provided one earlier. This is caused by recent changes to the API, and usages of the API in the sim haven't been updated. Early versions of BASE did not have this problem.

**Comment by jessegreenberg** [comment]
This was fixed as part of https://github.com/phetsims/scenery-phet/issues/364, and  the HTML is sound again (no `div`s, no `p`s, and `li`s are direct descendants of `ul`s).

![capture](https://user-images.githubusercontent.com/6396244/38587379-74c93712-3cde-11e8-83c1-bb67b4688cc3.PNG)

Closing.
